docs/guide/tutorials.rst

```diff
@@ -9,134 +9,199 @@ Structure of Data Sets
+		s7p.s7p_1.Heavy s7p.s7p_1.Light     peak_apex_int uM                                                                                            linear               1.88269                          1.93E-04                                                                                                     1.00E-15                               1.00E+15                               1.00E-15                               1.00E+15
+		\-              \-                  \-            \-                  \-   \-   \-   \-   \-                      \-                   \-       \-                   \-                               \-                                   \-                                  \-                                  \-                                     \-                                     \-                                     \-
+		=============== =================== ============= =================== ==== ==== ==== ==== ======================= ==================== ======== ==================== ================================ ==================================== =================================== =================================== ====================================== ====================================== ====================================== ======================================
+
+|
 
 #. sequence.csv
-	A list of injections are store in this file following the header schema :
-	``sample_name,sample_group_name,sequence_segment_name,sample_type,``
-	``original_filename,batch_name,rack_number,plate_number,pos_number,``
-	``inj_number,dilution_factor,inj_volume,inj_volume_units,operator_name,``
-	``acq_method_name,proc_method_name,acquisition_date_and_time,scan_polarity,``
-	``scan_mass_low,scan_mass_high``.
-	A sample entry may look like the following :
-	``170808_Jonathan_yeast_Sacc2_1x,group1,sequence1,Unknown,``
-	``170808_Jonathan_yeast_Sacc2_1x,BatchName,,,,2,,3,uL,,MethodName,,,,,``.
+	This file contains information about all the injections in the data set and their values,
+	fields such as `sample_name` and `original_filename` refer to the files names stored in the `mzML` folder.
+	The header including a sample entry is shown below:
+
+	.. table:: parameters.csv Headers
+		:widths: auto
+
+		============================== ================= ===================== =========== ============================== ========== =========== ============ ========== ========== =============== ========== ================ ============= =============== ================ ========================= ============= ============= ==============
+		sample_name                    sample_group_name sequence_segment_name sample_type original_filename              batch_name rack_number plate_number pos_number inj_number dilution_factor inj_volume inj_volume_units operator_name acq_method_name proc_method_name acquisition_date_and_time scan_polarity scan_mass_low scan_mass_high
```

Comment:
I would recommend adding a note here: "Information in the sequence that are not required for this workflow are left empty".  When you get to the FIA-MS workflow for example many of the blank entries here will need to be filled for the workflow to run properly.

Reply:
I have included the note, please let me know if anything else needs changing.